packages/bot-web-ui/src/components/run-panel.jsx

```diff
@@ -98,30 +98,45 @@ const drawerFooter = ({
     );
 };
 
-class RunPanel extends React.PureComponent {
+class RunPanel extends React.Component {
```

Comment:
What's the reason for changing this into a `Component`?

Reply:
> What's the reason for changing this into a `Component`?

this component has a constructor and state but after update dialog to modal it has been removed so I revert this change

Reply:
> > What's the reason for changing this into a `Component`?
> 
> this component has a constructor and state but after update dialog to modal it has been removed so I revert this change

Done